rotkehlchen/assets/utils.py

```diff
@@ -371,7 +371,15 @@ def get_crypto_asset_by_symbol(
         asset_type=asset_type,
         chain_id=chain_id,
     )
-    if len(assets_data) != 1:
+    if len(assets_data) < 1:
+        return None
+
+    if len(assets_data) > 1:
+        # check if all the assets are part of the same asset_collection
+        multiassets = set(GlobalDBHandler.get_assets_in_same_collection(assets_data[0].identifier))
+        if len(set(assets_data).difference(multiassets)) == 0:
+            return assets_data[0]
```

Comment:
this is bad because if for some reason the asset order changed in the db it will affect the events and it can lead to errors. I believe you added it due to the tron issue right? YOu can add a mapping for all the locations for this case